Enhance logs.md with syslog forwarding details (#495)
xcp-ng/xcp-ng-org: docs/guides/logs.md

`xensource.log` has many particular and different configuration parameters, so another `logrotate` config is used: `/etc/xensource/xapi-logrotate.conf` in a shell script `/opt/xensource/libexec/xapi-logrotate.sh` that executes `logrotate` with this specific config.

There is normally no need to run it manually, a cron task `/etc/cron.d/xapi-logrotate.cron` is present to schedule it each hour.
The goal of this special config is to keep the `xensource.log` files for one month, and to limit the number of log files to 100.

## Forward XCP-ng syslog to remote server
Syslog output can be forwarded to an external server such as Graylog or Wazuh for analysis, alerting and search capabilities.

> The following `xe` commands can be used from the XCP-ng CLI to enable syslog forwarding or this can be setup in Xen Orchestra on the Advanced tab for the host.

Reviewer: ```suggestion
The following `xe` commands can be used from the XCP-ng CLI to enable syslog forwarding. Alternatively, this can be set up in Xen Orchestra, on the host's **Advanced** tab.
```

```
xe host-param-set uuid=<host_uuid> logging:syslog_destination=<hostname or IP>:port
xe host-syslog-reconfigure host-uuid=<host_uuid>
```

Refer to this forum post for a high-level guide to setup Wazuh to receive and view syslog data:
[Guide: XCP-ng syslog forwarding to Wazuh](https://xcp-ng.org/forum/topic/12322/guide-xcp-ng-syslog-forwarding-to-wazuh).

> Refer to Graylog's documentation for setting up a syslog input.

Reviewer: Is there an external URL we can share for this documentation?

Author: Here is the external URL for the documentation update:
https://docs.xcp-ng.org/guides/logs/

Reviewer: How about we link to https://go2docs.graylog.org/current/home.htm?

Author: For Graylog, please use:
https://go2docs.graylog.org/current/getting_in_log_data/syslog_inputs.html

Reviewer: ```suggestion
Refer to [Graylog's documentation](https://go2docs.graylog.org/current/getting_in_log_data/syslog_inputs.html) for setting up a syslog input.
```
